src/poliastro/plotting.py

```diff
@@ -399,6 +327,95 @@ def savefig(self, filename, **layout_kwargs):
+
+    def __init__(self):
+        super().__init__()
+        self._layout = Layout(
+            autosize=True,
+            scene=dict(
+                xaxis=dict(
+                    title="x (km)",
+                ),
+                yaxis=dict(
+                    title="y (km)",
+                ),
+                zaxis=dict(
+                    title="z (km)",
+                ),
+                aspectmode="data",  # Important!
+            ),
+        )
+
+    @u.quantity_input(elev=u.rad, azim=u.rad, distance=u.km)
+    def set_view(self, elev, azim, distance=5 * u.km):
+        x = distance * np.cos(elev) * np.cos(azim)
+        y = distance * np.cos(elev) * np.sin(azim)
+        z = distance * np.sin(elev)
+
+        self._layout.update({
+            "scene": {
+                "camera": {
+                    "eye": {
+                        "x": x.to(u.km).value, "y": y.to(u.km).value, "z": z.to(u.km).value
+                    }
+                }
+            }
+        })
+
+    def _redraw_attractor(self, min_radius=0 * u.km):
+        # Select a sensible value for the radius: realistic for low orbits,
+        # visible for high and very high orbits
+        radius = max(self._attractor.R.to(u.km), min_radius.to(u.km))
+        if radius < self._attractor_radius:
+            # If the resulting radius is smaller than the current one, redraw it
+            shape = _plot_sphere(radius, BODY_COLORS.get(self._attractor.name, "#999999"), self._attractor.name)
+            # Overwrite stored properties
+            self._attractor_radius = radius
+            self._attractor_data = shape
+
+    def _plot_trajectory(self, trajectory, label, color, dashed):
+        trace = Scatter3d(
+            x=trajectory.x.to(u.km).value, y=trajectory.y.to(u.km).value, z=trajectory.z.to(u.km).value,
+            name=label,
+            line=dict(
+                color=color,
+                width=5,
+                dash='dash' if dashed else 'solid',
+            ),
+            mode="lines",  # Boilerplate
+        )
+        self._data.append(trace)
+
+    def plot(self, orbit, *, label=None, color=None):
+        """Plots state and osculating orbit in their plane.
+
+        Parameters
+        ----------
+        orbit : ~poliastro.twobody.orbit.Orbit
+        label : string, optional
+        color : string, optional
+        """
+        if color is None:
+            color = next(self._color_cycle)
+
+        self.set_attractor(orbit.attractor)
+
+        self._redraw_attractor(orbit.r_p * 0.15)  # Arbitrary threshold
+
+        label = _generate_label(orbit, label)
+        _, trajectory = orbit.sample()
+
+        self._plot_trajectory(trajectory, label, color, True)
+
+        # Plot required 2D/3D shape in the position of the body
+        radius = min(self._attractor_radius * 0.5, (norm(orbit.r) - orbit.attractor.R) * 0.3)  # Arbitrary thresholds
+        shape = _plot_sphere(radius, color, label, center=orbit.r)
```

Comment:
Exactly the same case: both `plot` methods are identical except for one function. Renaming `_plot_sphere` to `OrbitPlotter3D._plot_sphere`, we could move `plot` to the base class.

Reply:
Yes True! Done!